web/src/AutoBoundariesMode.svelte

```diff
@@ -136,21 +164,68 @@
             "orange",
             "black",
           ],
-          "fill-opacity": hoverStateFilter(0.3, 0.7),
+          "fill-opacity": hoverStateFilter(0.2, 0.4),
         }}
-        on:click={add}
+        layout={{
+          visibility: selectedPrioritization == "none" ? "visible" : "none",
+        }}
+        manageHoverState
         hoverCursor="pointer"
-      />
+        on:click={add}
+      >
+        <!-- 
```

Comment:
https://github.com/dimfeld/svelte-maplibre/blob/master/src/lib/Popup.svelte
The new version using Svelte 5 fixes this, I think, by making a generic type for the component. (I don't think the fix depends on Svelte 5, it was just made during the migration.)  I haven't had a proper chance to play around with it, so I don't know how to specify the type parameter, but I agree something like this would be so awesome to have.